connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorCheckpointConnector.java

```diff
@@ -133,6 +132,13 @@ public Class<? extends Task> taskClass() {
     // divide consumer groups among tasks
     @Override
     public List<Map<String, String>> taskConfigs(int maxTasks) {
+        if (Objects.isNull(knownConsumerGroups)) {
```

Comment:
Can we just use `== null`? `Objects::isNull` was introduced in Java 8 to make streams operations and other lambda/method reference-based APIs easier to work with.

See the [Javadocs](https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#isNull-java.lang.Object-):
> API Note:
This method exists to be used as a [Predicate](https://docs.oracle.com/javase/8/docs/api/java/util/function/Predicate.html), `filter(Objects::isNull)`